tests/main/interfaces-udev/task.yaml

```diff
@@ -0,0 +1,31 @@
+summary: Ensure that the udev interface backend works.
+
+details: |
+    This test checks that the udev rules file is created when a snap declaring
+    a dependency on an interface (being it a slot or a plug) and that concrete
+    dependency has a related udev snippet, then the udev rules files are created
+    on install and removed after it is uninstalled.
+
+    Currently the ony interface that declares a udev snippet is the modem-manager
+    interface for its slot. This test can be easily extended with variants when
+    more interfaces declare udev snippets.
+
+prepare: |
+    echo "Given a snap declaring a slot with associated udev rules is installed"
+    snapbuild $TESTSLIB/snaps/modem-manager-consumer .
+    snap install modem-manager-consumer_1.0_all.snap
+
+restore: |
+    rm -f modem-manager-consumer_1.0_all.snap
+
+execute: |
+    echo "Then the udev rules files specific to it are created"
+    test -f /etc/udev/rules.d/70-snap.modem-manager-consumer.rules
```

Comment:
Can we check some content here.. doesn't have to be precise as we have unit tests for that. We just want to tell that this file has a rule of whatever kind instead of being empty.